### PR TITLE
Operations documentation

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: doc/user/deploying.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Deploying TruffleRuby
+
+If you're attempting to experiment with deploy TruffleRuby to production we'd
```
Comment (aardvark179) [inline doc/user/deploying.md]: 'deploying' not 'deploy'

File: doc/user/deploying.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+important to understand the different configurations of TruffleRuby, as each has
+different capabilities and performance characteristics. You should pick the
+execution mode that is appropriate for your application.
+
+When distributed as part of GraalVM, TruffleRuby by default runs in the *native*
+configuration. In this configuration, TruffleRuby is ahead-of-time compiled to a
+standalone native executable. This means that you don't need a JVM installed on
+your system to use it. The advantage of the native configuration is that it
+[starts about as fast as MRI](../contributor/svm.md), it may use less memory,
+and it becomes fast in less time. The disadvantage of the native configuration
```
Comment (aardvark179) [inline doc/user/deploying.md]: I'd add "than the JVM configuration," as on my first read I thought you were claiming lower memory than MRI.

File: doc/user/deploying.md (hunk 1 above)
Comment (chrisseaton) [reply doc/user/deploying.md]: Fixing.

File: doc/user/deploying.md (hunk 2 above)
Comment (chrisseaton) [reply doc/user/deploying.md]: Fixing.

**Pull request merged**